test/libsolidity/syntaxTests/constants/cyclic_dependency_file_constants.sol

```diff
@@ -0,0 +1,13 @@
+==== Source: A.sol ====
+import "B.sol";
+
+uint256 constant A = B + 1;
+
+==== Source: B.sol ====
+import "A.sol";
+
+uint256 constant B = A + 1;
```

Comment:
The fact that these are in separate files does not seem relavant to the test. It would still fail if it was a single file, wouldn't it?

Reply:
Right, I guess for such error case we can just put things as single files. Updated.

Reply:
Fair enough.